src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,18 @@
-import React from 'react';
+import { Posts } from '../../types/Posts';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Posts[];
+};
+
+export const PostList: React.FC<Props> = ({ posts = [] }) => (
```

Comment:
it's extra, bc you put posts: Posts[] in your Props, it means that you never have a case, that you don't have posts
```suggestion
  posts: Posts[];
};

export const PostList: React.FC<Props> = ({ posts }) => (
```